### PR TITLE
Re-indent AAQ view and Question.auto_tag to use longer lines.

### PR DESCRIPTION
**Pull request by mythmon**
I was reading through these functions for [Bug 1195859](https://bugzilla.mozilla.org/show_bug.cgi?id=1195859). It made it easier to read through by re-indenting them to take advantage of longer lines, so I did that.

It followed me home, can we keep it?

### CODE REVIEW

File: kitsune/questions/models.py [hunk 1]
```diff
@@ -244,18 +244,17 @@ def auto_tag(self):
         You don't need to call save on the question after this.
 
         """
-        to_add = (self.product_config.get('tags', []) +
-                  self.category_config.get('tags', []))
-
+        to_add = (self.product_config.get('tags', []) + self.category_config.get('tags', []))
```
Comment (willkg) [inline kitsune/questions/models.py]: Nit: Now that this is on one line, you can ditch the parens if you want.

**Comment by willkg** [comment]
This looks fine to me. r+!

File: kitsune/questions/models.py (hunk 1 above)
Comment (mythmon) [inline kitsune/questions/models.py]: Ah, yes. I'll do that. I basically ran around hitting `J` in vim.

**Comment by mythmon** [comment]
/me waits for Travis, and then hits the :green_apple: button.

**Pull request merged**